code/api/src/modules/user/types.js

```diff
@@ -46,13 +46,13 @@ const UserGenderType = new GraphQLObjectType({
 })
 
 // User Orders type
-const UserOrdersType = new GraphQLObjectType({
-  name: 'userOrders',
-  description: 'User Order Type',
-
-  fields: () => ({
-    orders: { type: new GraphQLList( OrderType )}
-  })
-})
-
-export { UserType, UserLoginType, UserGenderType, UserOrdersType }
+// const UserOrdersType = new GraphQLObjectType({
```

Comment:
Just want to make a note that there is commented out code. No worries if it is for future reference!